### PR TITLE
edit profile component 

### PR DESCRIPTION
**Pull request by jose2a**
the component's CSS file and HTML files were changes the fit the new style for the UI.

### CODE REVIEW

File: src/app/components/pages/user-management/profile/profile.component.html [hunk 1]
```diff
@@ -1,122 +1,107 @@
...
+            Last name is <strong>required</strong>
+          </mat-error>
+        </mat-form-field>
+      </div>
+    </div>
+​
+    <div class="container">
+        <mat-form-field>
+          <input matInput readonly=true id="inputEmail" placeholder="Email" formControlName="email"
+            (blur)=formFilled() (mouseleave)=formFilled() [pattern]="emailPattern"><!--required>
```
Comment (jurczakn) [inline src/app/components/pages/user-management/profile/profile.component.html]: Remove commented out code.

File: src/app/components/pages/user-management/profile/profile.component.html [hunk 2]
```diff
@@ -1,122 +1,107 @@
...
+          <mat-error *ngIf="form.controls.email.errors?.RevatureEmail">
+            Must end with <strong>@revature.com</strong>
+          </mat-error>
+          -->
+        </mat-form-field>
+    
+      <div>
+        <mat-form-field>
+          <input matInput readonly=true id="inputUsername" placeholder="Username" formControlName="username"
+            (blur)=formFilled() (mouseleave)=formFilled()> <!--required>
```
Comment (jurczakn) [inline src/app/components/pages/user-management/profile/profile.component.html]: More commented out code.

**Review by jurczakn — CHANGES_REQUESTED** [review]

File: src/app/components/pages/user-management/profile/profile.component.html (hunk 1 above)
Comment (jose2a) [reply src/app/components/pages/user-management/profile/profile.component.html]: We removed the comments

**Review by jurczakn — APPROVED** [review]
Good

**Pull request merged**